pkgs/by-name/ra/radarr/package.nix

```diff
@@ -0,0 +1,203 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  sqlite,
+  withFFmpeg ? true, # replace bundled ffprobe binary with symlink to ffmpeg package.
+  ffmpeg,
+  fetchYarnDeps,
+  yarn,
+  fixup-yarn-lock,
+  nodejs,
+  nixosTests,
+  # update script
+  writers,
+  python3Packages,
+  nix,
+  prefetch-yarn-deps,
+  fetchpatch,
+  applyPatches,
+}:
+let
+  version = "5.19.2.9720";
```

Comment:
custom builds from PR branches are not supported; do not fake the version number for custom builds especially of unreleased versions